languages/python/reference/README.md

```diff
@@ -538,3 +538,181 @@ These will very rarely be encountered in the wild, the first because it's more o
+| [except][keyword-except] | [ ]() | [ ]() | 
+| [Exceptions][exceptions-general]| [ ]() | [ ]() | 
+| [Exception catching][exception-catching] | [ ]() | [ ]() | 
+| [Exception handling][exception-handling]| [ ]() | [ ]() | 
+| [Exception hierarchy][exception-hierarchy]| [ ]() | [ ]() | 
+| [Exception message][exception-message]| [ ]() | [ ]() | 
+|[Exiting loops][]| [ ]() | [ ]() | 
+| [Expressions][expressions] | [ ]() | [ ]() | 
+| [False][keyword-false] | [ ]() | [ ]() | 
+| [filter][builtin-functions-filter] `functional` |  []() | []() | 
+| [finally][keyword-false] | [ ]() | [ ]() | 
+| [float][builtin-types-float]`basic`|   [#1026](https://github.com/exercism/v3/issues/1026)         |         |
+| [for][keyword-for] | [ ]() | [ ]() | 
+| [for loops][for-loops] | [ ]() | [ ]() | 
+| [frozenset][builtin-types-frozenset] `intermediate`| [ ]() | [ ]() |
+| [from][keyword-from] | [ ]() | [ ]() | 
+| [Functions][functions-general] | [ ]() | [ ]() | 
+| [Function Definition][function-definition] | [ ]() | [ ]() | 
+|[Function Signature][function-signature] | [ ]() | [ ]() | 
+|[Generators comprehension][generator-comprehension]| [ ]() | [ ]() |
+|[Generators][generators]| [ ]() | [ ]() |
+| [global][keyword-global] | [ ]() | [ ]() |
+| [Higher-order functions][higher-order-functions] `functional` | [ ]() | [ ]() |
+| [Identity testing][identity-testing]| [ ]() | [ ]() |
+| [if][keyword-if] | [ ]() | [ ]() |
+| [Immutability][immutability] `functional` | [ ]() | [ ]() |
+| [Immutability in Python][immutability] | [ ]() | [ ]() |
+| [implicit self][implicit-self] | [ ]() | [ ]() |
+| [importing][importing] | [ ]() | [ ]() |
+| [in][keyword-in] | [ ]() | [ ]() |
+| [Indexing][indexing] | [ ]() | [ ]() |
+| [Inequality][inequality] | [ ]() | [ ]() | 
+| [Inheritance][inheritance-general] | [ ]() | [ ]() | 
+| [Instance Attributes][instance-attributes] | [ ]() | [ ]() | 
+| [Instance Methods][instance-methods] | [ ]() | [ ]() | 
+| [Instance Properties][instance-properties] | [ ]() | [ ]() | 
+| [Instantiation][instantiation] | [ ]() | [ ]() | 
+| [Initialization][initialization] | [ ]() | [ ]() | 
+| [int][builtin-types-int] `basic`|   [#1025](https://github.com/exercism/v3/issues/1025)         |         |
+| [Integer comparison][integer-comparison]| [ ]() | [ ]() |
+| [Interfaces][integer-comparison] | [ ]() | [ ]() |
+| [Iterators][iterators] | [ ]() | [ ]() |
+| [Iterables][iterables] | [ ]() | [ ]() |
+| [Iteration][iteration] | [ ]() | [ ]() |
+| [is][keyword-is] | [ ]() | [ ]() |
+| [Keyword parameters][keyword-parameters]| [ ]() | [ ]() |
+| [Keyword-only parameters][keyword-only-parameters]| [ ]() | [ ]() |
+|[`**kwargs`][star-star-kwargs] | [ ]() | [ ]() |
+| [lambda][keyword-lambda] `intermediate`|   [ ]()         |         |
+| [list][builtin-types-list] `basic`|   [#859] (https://github.com/exercism/v3/issues/859)        |         |
+| [list comprehension][list-comprehension] | [ ]() | [ ]() |
```

Comment:
There's a link reference for `list-comprehension` here, but there's no corresponding link entry ... we'll need to add `[list-comprehension]: ./concepts/list_comprehension.md` in the appropriate location in the link entries.